bril-rs/brillvm/src/llvm.rs

```diff
@@ -1271,13 +1271,20 @@ pub fn create_module_from_program<'a>(
 
                 let llvm_func = runtime_module.add_function(func_name, ty, None);
                 args.iter().zip(llvm_func.get_param_iter()).for_each(
-                    |(Argument { name, .. }, bve)| match bve {
-                        inkwell::values::BasicValueEnum::IntValue(i) => i.set_name(name),
-                        inkwell::values::BasicValueEnum::FloatValue(f) => f.set_name(name),
-                        inkwell::values::BasicValueEnum::PointerValue(p) => p.set_name(name),
-                        inkwell::values::BasicValueEnum::ArrayValue(_)
-                        | inkwell::values::BasicValueEnum::StructValue(_)
-                        | inkwell::values::BasicValueEnum::VectorValue(_) => unreachable!(),
+                    |(Argument { name, .. }, bve)| {
+                        use inkwell::values::BasicValueEnum::{
+                            ArrayValue, FloatValue, IntValue, PointerValue, ScalableVectorValue,
+                            StructValue, VectorValue,
+                        };
```

Comment:
Would this perhaps be slightly more legible with just plain old `use inkwell::values::BasicValueEnum` and then qualifying the matches with just the enum name? Anyway, I guess that's pretty small potatoes.